pkg/asset/manifests/openshift.go

```diff
@@ -72,6 +72,7 @@ func (o *Openshift) Dependencies() []asset.Asset {
 		&openshift.BaremetalConfig{},
 		new(rhcos.Image),
 		&openshift.AzureCloudProviderSecret{},
+		&openshift.IronicVirtualMediaTLS{},
```

Comment:
I wonder if we can hide this inside of our BaremetalConfig (maybe we cannot).

Reply:
I have removed this asset now